packages/connections/src/components/connection-list/connection-menu.spec.tsx

```diff
@@ -174,4 +245,54 @@ describe('ConnectionMenu Component', function () {
       });
     });
   });
+  describe('function calls', function () {
+    it('should call the removeConnection function', function () {
+      const connectionInfo: ConnectionInfo = {
+        id: 'test-id',
+        favorite: {
+          name: 'First Server',
+        },
+        connectionOptions: {
+          connectionString: 'mongodb://kaleesi',
+        },
+      };
+      const mockDuplicateConnection = sinon.fake.resolves(null);
+      const mockRemoveConnection = sinon.fake.resolves(null);
+      render(
+        <ConnectionMenu
+          connectionString={'mongodb://kaleesi'}
+          duplicateConnection={mockDuplicateConnection}
+          removeConnection={mockRemoveConnection}
+          connectionInfo={connectionInfo}
+          iconColor="#EAEAEA"
+        />
+      );
+
+      fireEvent(
+        screen.getByRole('button'),
+        new MouseEvent('click', {
+          bubbles: true,
+          cancelable: true,
+        })
+      );
+      const duplicateConnectionButton = screen.getByText('Duplicate');
+      fireEvent(
+        duplicateConnectionButton,
+        new MouseEvent('click', {
+          bubbles: true,
+          cancelable: true,
+        })
+      );
+      const removeConnectionButton = screen.getByText('Remove');
+      fireEvent(
+        removeConnectionButton,
+        new MouseEvent('click', {
+          bubbles: true,
+          cancelable: true,
+        })
+      );
+      expect(mockDuplicateConnection.called).to.equal(true);
+      expect(mockRemoveConnection.called).to.equal(true);
```

Comment:
How do you feel about breaking these into separate tests? I think that'll ensure down the line one action doesn't accidentally call the other action.